x-pack/plugins/secops/public/components/timeline/data_providers/provider_item_actions.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  // @ts-ignore
```

Comment:
This `ts-ignore` shouldn't be needed with `EuiContextMenu` anymore. Other parts of the code no longer have it.